### DISCUSSION TITLE
It is now impossible to checkout the Helmfile source on Windows

### DISCUSSION THREAD

**Discussion by philomory** [helmfile/helmfile#199] [Questions] [answered]
The path `pkg/app/testdata/testapply/helm2:_upgrade_when_tns1` (specifically, the `:` in the path) is not valid on Windows, and means that it is now impossible to checkout this repo onto a Windows machine.

Obviously, Windows isn't the primary development environment for Helmfile, but, would it be possible to consider substituting a different character than `:` in that path? Or perhaps simply removing it, making `pkg/app/testdata/testapply/helm2_upgrade_when_tns1`?

**Comment by yxxhero** [comment]
@philomory I will think about it.

**Comment by philomory** [comment, marked as answer]
For reference, here's a PR to address the issue, if desired: https://github.com/helmfile/helmfile/pull/200

**Comment by ankitm123** [comment]
It would be good to have a new release with this fix, the `:` also breaks go mod tidy in linux when using 0.145.0:
```
create zip: pkg/app/testdata/testapply/helm2:_upgrade_when_tns1/foo_needs_tns2/bar/log: malformed file path "pkg/app/testdata/testapply/helm2:_upgrade_when_tns1/foo_needs_tns2/bar/log": invalid char ':'
```
